src/main/java/net/quedex/api/user/TimeTriggeredBatchAdded.java

```diff
@@ -0,0 +1,44 @@
+package net.quedex.api.user;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+public class TimeTriggeredBatchAdded {
```

Comment:
All names should start with `Timer...` instead of `TimerTriggeredBatch...`

Reply:
Changed.